pkg/chain/ethereum/ethereum_test.go

```diff
@@ -155,11 +155,11 @@ func TestPackTicket(t *testing.T) {
 		return bigInt
 	}
 
-	ticketValue := toBigInt("77475267169740498967948014258679832639111923451618263020575217281118610489031")
+	ticketValue := [8]byte{255, 255, 255, 255, 255, 255, 255, 255}
 	stakerValue := toBigInt("471938313681866282067432403796053736964016932944")
```

Comment:
Let's get rid of `ticketValue` and `stakerValue` from here and inline them in test cases below.